src/structures/Player.ts

```diff
@@ -139,15 +146,37 @@ export class Player {
   ): Promise<SearchResult> {
     return this.manager.search(query, requester);
   }
+  /**
+   * Move the player to another connected node
+   * @param name
+   */
+  async movePlayer(name: string) {
+    const node = this.manager.nodes.get(name)
+    if(!node.connected) throw Error('The node is not connected');
+    if (!name || node.options.identifier === this.node.options.identifier) return this;  
```

Comment:
The left hand side condition should be before accessing the Node members and the name parameter would be the [identifier](https://github.com/MenuDocs/erela.js/blob/master/src/structures/Node.ts#L118), simply:
```javascript
if (this.node.options.identifier === name) {}
```